src/main/resources/org/jenkinsci/plugins/workflow/steps/WaitForConditionStep/config.jelly

```diff
@@ -24,4 +24,15 @@ THE SOFTWARE.
 -->
 
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core"/>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="minRecurrencePeriod" title="${%Minimum Recurrence Period}">
+        <f:number clazz="positive-number"/>
+    </f:entry>
+    <f:entry field="maxRecurrencePeriod" title="${%Maximum Recurrence Period}">
+        <f:number clazz="positive-number"/>
```

Comment:
You need a `default` here, which would most easily be a constant in `DescriptorImpl`, moved there from the `Step` itself. [Documentation](https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md#handling-default-values)